GitUI/CommandsDialogs/RevisionDiff.cs

```diff
@@ -42,6 +43,10 @@ public RevisionDiff()
             DiffText.Font = AppSettings.DiffFont;
 
             GotFocus += (s, e) => DiffFiles.Focus();
+            Load += (s, e) =>
+            {
+                _revisionDiffController = new RevisionDiffController(Module);
```

Comment:
Passing Module object does not make sense, it is not updated when the RevisionDiff Module is updated

Reply:
> it is not updated when the RevisionDiff Module is updated

I've checked it, `Module` always contains the current information when switching between git repos.

Reply:
The working for was not updated for me.